petastorm/unischema.py

```diff
@@ -215,13 +217,15 @@ def make_namedtuple_tf(self, *args, **kargs):
         return self._get_namedtuple()(*args, **kargs)
 
     @classmethod
-    def from_arrow_schema(cls, parquet_dataset):
+    def from_arrow_schema(cls, parquet_dataset, warn_only=False):
```

Comment:
How about  `warn_only` -> `omit_unsupported_fields` (or some variant of the name to indicate which warnings are you referring to ) ?

Reply:
good idea, that will be much clearer.